src/ctia/bulk/core.clj

```diff
@@ -81,9 +80,12 @@
   "
   [func bulk & args]
   (->> bulk
-       (pmap (fn [[entity-type entities]]
-               [entity-type
-                (apply func entities (entity-type-from-bulk-key entity-type) args)]))
+       (pmap (fn [[bulk-key entities]]
```

Comment:
Would you mind changing `bulk-key` by `bulk-k` to avoid confusion with the function?